server/integration_tests/test_data/translate_hindi/query_1/chart_config.json

```diff
@@ -3,230 +3,6 @@
     "categories": [
       {
         "blocks": [
-          {
```

Comment:
@keyurva these seem related to translation, do you have any ideas what could be causing these?

Reply:
Thanks for updating the goldens. Unfortunately the translations returned by the api can vary and the goldens will need to be updated when they do.

Reply:
Is the failure non-determinism vs. some sporadic model change? (if the former, we should consider checking for select keywords instead of full strings)

Reply:
I don't think it'll be sporadic. I've filed b/307967463 to address it.